js/index.js

```diff
@@ -1 +1,77 @@
 // Your code goes here
+
+funBusHead = document.querySelector('.logo-heading');
+navBar = document.querySelector('.nav');
+navItems = document.querySelectorAll('.nav-link');
+headerH2 = document.querySelector('.intro h2');
+allParagraphs = document.querySelectorAll('p');
+allExpeditionsTitle = document.querySelectorAll('.destination h4');
+footer = document.querySelector('.footer');
+
+funBusHeading.addEventListener('mouseover', function () {
+    this.classList.add('funBusHeading');
+});
+
+funBusHead.addEventListener('mouseleave', function () {
+    this.class.remove('funBusHeading');
+});
+
+navBar.addEventListener('keydown', function () {
+    event.preventDefault();
+    this.classList.add('navBar');
+    console.log('navBar');
+    this.addEventListener('animationend', myEndFunction);
+});
+
+function myEndFunction() {
+    this.classList.remove('navBar');
```

Comment:
There doesn't exist a class called `navBar` to remove. `navBar` is not defined in your less/css